packages/legend-query-builder/src/components/explorer/QueryBuilderPropertySearchPanel.tsx

```diff
@@ -177,81 +286,153 @@ const QueryBuilderTreeNodeViewer = observer(
       }
       return childNodes;
     };
+
+    const handleHighlightNode = (): void => {
+      explorerState.propertySearchState.setIsSearchPanelOpen(false);
+      explorerState.propertySearchState.resetSearch();
+      explorerState.highlightTreeNode(node.id);
+    };
+
     const parentNode = propertySearchState.indexedExplorerTreeNodes.find(
       (pn) =>
         node instanceof QueryBuilderExplorerTreePropertyNodeData &&
         node.parentId === pn.id,
     );
+
     const propertyName =
       parentNode instanceof QueryBuilderExplorerTreeSubTypeNodeData
-        ? prettyPropertyNameForSubType(node.id)
-        : node instanceof QueryBuilderExplorerTreeSubTypeNodeData
-          ? prettyPropertyNameForSubTypeClass(node.id)
-          : prettyPropertyNameFromNodeId(node.id);
+        ? prettyPropertyNameForSubType(node.id, true)
+        : prettyPropertyNameFromNodeId(node.id, true);
+
+    const nodeExpandIcon = isExpandable ? (
+      node.isOpen ? (
+        <ChevronDownIcon />
+      ) : (
+        <ChevronRightIcon />
+      )
+    ) : (
+      <div />
+    );
+
+    const docText: string | null =
+      node instanceof QueryBuilderExplorerTreePropertyNodeData
+        ? (node.property.taggedValues.find(
+            (taggedValue) =>
+              taggedValue.tag.ownerReference.value.path ===
+                CORE_PURE_PATH.PROFILE_DOC &&
+              taggedValue.tag.value.value === PURE_DOC_TAG,
+          )?.value ?? null)
+        : null;
+    const formattedDocText =
+      docText !== null
+        ? formatTextWithHighlightedMatches(
+            docText,
+            propertySearchState.searchText,
+            'query-builder-property-search-panel__node__doc',
```

Comment:
Could we also create a variable for this too just like `QUERY_BUILDER_EXPLORER_SEARCH_INPUT_NAME` ?

Reply:
Done!